nodejs/README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Steps
 
-1. Start a locally running `mongod` version >= 4.2 running on port 27017
-2. Install dependencies with npm
+1. Clone this repository.
+2. Navigate to the nodejs folder.
+
+   ```sh
+   cd nodejs
+   ```
+
+3. Start a locally running `mongod` version >= 4.2 running on port 27017
```

Comment:
We should probably clarify Enterprise across the READMEs, or link back to a document that provides the server requirements.